Cargo.toml

```diff
@@ -65,8 +65,11 @@ utxo = { path = "utxo"}
 [profile.release]
 debug-assertions = true
 overflow-checks = true
+lto = true
 
 [profile.test]
-opt-level = 2
 debug-assertions = true
 overflow-checks = true
+
+[profile.test.package.script]
+opt-level = 2
```

Comment:
Does this inherit the other properties of the "global" test profile?

Reply:
I think so. I only tested that script now is release.

Reply:
Ok docs seem to suggest so, although it's not super explicit. Let's wait and see how long the CI takes.